core/tests/protractor.conf.js

```diff
@@ -44,6 +44,10 @@ var suites = {
     explorationFeedbackTab: [
       'protractor_desktop/explorationFeedbackTab.js'
     ],
+    
+    explorationMarkCorrect: [
```

Comment:
Is this still required? I think all the e2e tests are failing because of this PTAL!

Reply:
Removed